com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/vivian-ng/telsis_translator_flutter/releases/download/0.2.1/telsis-translator-flutter_0.2.1.tar.gz
```

Comment:
As I assume this is a pre-built binary, you should limit it to x86_64: https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
I have deleted the desktop, metainfo, and icon files. Plus added the `flathub.json` file to limit to x86_64.